doc/ss_logger.rst

```diff
@@ -0,0 +1,172 @@
+***************
+SmartSim Logger
+***************
+========
+Overview
+========
+SmartSim supports experiment tracking through logging functionality
+offered by the SmartSim `log` module. The logger, supported by Python logging, enables
+monitoring the experiment at runtime by allowing users to print messages **to stdout**
+and/or **to file** from within the Python driver script. The SmartSim logger permits users to categorize
+messages by severity level. Users may instruct SmartSim to print to stdout
```

Comment:
Beginning with `Users may`, this should be a new paragraph. Also, replace `to print to stdout` with `to log`